### PR TITLE
 pythonPackages.snapcast: init at 2.0.9 

### PR DESCRIPTION
**Pull request by peterhoeg**
###### Motivation for this change

HA supports snapcast!

Cc @dotlambda 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/snapcast/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0hlfcg0qdfavjzhxz4v2spjkj6440a1254wrncfkfkyff6rv9x3s";
+  };
+
+  propagatedBuildInputs = [ construct ];
+
+  # I need to see why the tests are not running
+  doCheck = false;
```
Comment (dotlambda) [inline pkgs/development/python-modules/snapcast/default.nix]: Did you try
```
checkPhase = ''
  py.test
'';
```
?

Comment (peterhoeg) [reply pkgs/development/python-modules/snapcast/default.nix]: No tests in tarball. https://github.com/happyleavesaoc/python-snapcast/issues/23

**Comment by dotlambda** [comment]
From https://github.com/happyleavesaoc/python-snapcast:

> Control Snapcast in Python **3**

**Comment by peterhoeg** [comment]
Not sure if setting ```meta.broken``` really communicates the proper intent here. It's not that it's broken as much as it isn't supported which really is two different things.

**Comment by dotlambda** [comment]
We do this by `disabled = !isPy3k` for python packages :)

**Comment by peterhoeg** [comment]
@GrahamcOfBorg eval

**Comment by peterhoeg** [comment]
@GrahamcOfBorg build python2Packages.snapcast python3Packages.snapcast
@GrahamcOfBorg test home-assistant

**Pull request merged**